src/Program.cs

```diff
@@ -11,6 +11,14 @@
 var builder = WebApplication.CreateBuilder(args);
 
 // Add services to the container.
+bool ConsoleOptions = builder.Configuration.GetSection("ConsoleOptions").GetValue<bool>("Show logs");
+bool GamePromptOptions = builder.Configuration.GetSection("GamePromptOptions").GetValue<bool>("Open Game");
+
+// shows information about loading assets/api stuff
+if (ConsoleOptions == true)
+{
+    VerboseLogging.log();
```

Comment:
This is wrong practice - the class responsible for logs should hold information about the logging level and print messages depending on it, and the messages should be in the place of use and not in the class responsible for logging.

So for example here should be:
```
MyLogger.set_level( log_level_from_config );
MyLogger.log("Starting SoDOff API server", MyLogger.INFO);
```

and in some other places only log message - for example: `MyLogger.log("Can't read item database", MyLogger.ERROR);`